cmd/spire-server/cli/run/run.go

```diff
@@ -465,6 +451,10 @@ func validateConfig(c *config) error {
 		}
 	}
 
+	if c.Server.DeprecatedSVIDTTL != "" {
+		return errors.New(`the "svid_ttl" configurable has been deprecated and renamed to "default_svid_ttl"; please update your configuration`)
```

Comment:
I think we have built-in detection for invalid configurables, so we can probably remove this check (and the DeprecatedSVIDTTL struct field) altogether?

Reply:
You are right! I didn't noticed that validation, will fix

Reply:
Right now we only log a warning when undefined configurables are used, with a note saying that it will be an error in a future release. If we added that warning at or before 0.9.0 we could also change that to an error for this release. If it was added after 0.9.0, we'll have to leave it as a warning.

Reply:
@marcosy, would you be willing to figure out when we added the warning for undefined configurables so we can determine if we can change it to an error?

Reply:
sure! I'll check

Reply:
We added it at v0.9.1